sample-workflows/transifex-pull.yml

```
@@ -32,14 +34,15 @@ jobs:
         with:
           ref: ${{ matrix.version }}
           fetch-depth: 0
-      - run: curl https://raw.githubusercontent.com/python-docs-translations/transifex-automation/master/sample-workflows/transifex-util.py
-      - run: ./manage_translation.py recreate_tx_config --language XX --project-slug python-newest --version 3.14
+      - run: curl -O https://raw.githubusercontent.com/python-docs-translations/transifex-automation/master/sample-workflows/transifex-util.py
```

We renamed the repo to transifex-automations, let's reduce redirects

```suggestion
      - run: curl -O https://raw.githubusercontent.com/python-docs-translations/transifex-automations/master/sample-workflows/transifex-util.py
```